lakefile.lean

```diff
@@ -75,14 +75,14 @@ post_update pkg do
   https://github.com/leanprover/lean4/issues/2752
   -/
   let wsToolchainFile := rootPkg.dir / "lean-toolchain"
-  let mathlibToolchain ← IO.FS.readFile <| pkg.dir / "lean-toolchain"
+  let mathlibToolchain := (← IO.FS.readFile <| pkg.dir / "lean-toolchain").trim
   IO.FS.writeFile wsToolchainFile mathlibToolchain
```

Comment:
Let's *not* remove the newline from the file itself, for the sake of consistency

Reply:
Oops, thanks.